tyr/servers/solr/node.py

```diff
@@ -0,0 +1,72 @@
+from tyr.servers.server import Server
+
+
+class SolrNode(Server):
+
+    SERVER_TYPE = 'solr'
+    CHEF_RUNLIST = ['role[RoleSolr]']
+    IAM_ROLE_POLICIES = [
+        'allow-volume-control',
+        'allow-get-solr-schema'
+    ]
+
+    def __init__(self, group=None, server_type=None, instance_type=None,
+                 environment=None, ami=None, region=None, role=None,
+                 keypair=None, availability_zone=None, security_groups=None,
+                 block_devices=None, chef_path=None, subnet_id=None,
+                 dns_zones=None, is_master=True, is_slave=False,
```

Comment:
do we really need a is_master and is_slave? There's no 3rd option is there? If is_master = false isn't it a slave?